### ISSUE TITLE
Design Intent - Accessibility Feedback - MHV Medical Records, Phase 1 Radiology Flow/Images

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#96841]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the delayed written feedback! I was out of office for a few days last week and am finally catching up. I apologize if my slowness impacted your schedule.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should (if possible):** If there's any programmatic way of generating even basic descriptions of the images from the meta data you get, try to provide alt text for each image that describes what's shown. Even something like "Left foot, anterior" or something similarly short would be good.
   - [ ] **Coding note:** If it's not possible to generate meaningful alt text, I think something like "1 of 16" would be appropriate here --- although I reserve the right to change my mind at your midpoint as I talk to more accessibility colleagues! 😆 My thinking: Without a description of the image, the index may still be useful, eg. if your doctor tells you that the fracture is most clear in image 7 and you want to share it with a sighted family member. Having alt text that's duplicative of the onscreen label for each image isn't ideal, but it's the least-bad option here.
- [ ] **Should:** I'd encourage you to use an [On This Page component](https://design.va.gov/components/on-this-page) on both the test details page (helps with discoverability of the "Images" section) and the images page (helps with discoverability of the "Download all images" section).
- [ ] **Consider:** A regular person isn't going to be able to view DICOM files, but they may have family or friends who are medical professionals that they want to share the images with. Even if the images displayed aren't high enough resolution to be useful in a clinical context, consider whether it makes sense to support some other way of viewing the images besides a square thumbnail. That may include an option to download low-res JPEGs in addition to the DICOM files, or linking the square thumbnail to a larger (if still low-res) view of the image not constrained by the requirements of a square thumbnail.
- [ ] **Must:** On the [test details page](https://www.figma.com/design/SGP1z2LejUWqDZyT61po5J/Medical-Records---Phase-1?node-id=11403-51895&t=3nyUJMy9X1Ukp95d-4) in the "Images" section, include some text warning users about (1) potential long download times (and indicating how they'll be notified), and (2) how long they will be able to view the images from this request. Many users, but especially users with disabilities that make using a computer more difficult, will benefit from that context to help them decide if it's a good time to push that request button or if they should wait until later.
- [ ] **Coding note:** In cases where the image request goes quickly (only a handful of images, maybe?) some kind of `aria-live` region may be needed to announce that they're ready to screen reader users. I'll include more notes on this at your midpoint based on how your design work evolves.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
